SIGSEGV on arena get sparkjob --type sparkjob

```bash
root@root1-HP-EliteBook-840-G2:/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena# ./bin/arena get sparkjob --type sparkjob
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0xfa4715]

goroutine 1 [running]:
github.com/kubeflow/arena/cmd/arena/commands.GetJobRealStatus(0x0, 0x0, 0x1, 0xfcd176)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/cmd/arena/commands/get.go:308 +0x55
github.com/kubeflow/arena/cmd/arena/commands.printJobSummary(0x1394f60, 0xc00038c0b0, 0x0, 0x0)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/cmd/arena/commands/get.go:265 +0x4d
github.com/kubeflow/arena/cmd/arena/commands.printSingleJobHelper(0x0, 0x0, 0xc000481b00, 0x0, 0x0)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/cmd/arena/commands/get.go:221 +0xdd
github.com/kubeflow/arena/cmd/arena/commands.printTrainingJob(0x0, 0x0, 0x7ffc2801d700, 0x0, 0x0)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/cmd/arena/commands/get.go:213 +0x92
github.com/kubeflow/arena/cmd/arena/commands.NewGetCommand.func1(0xc000174500, 0xc00026cf60, 0x1, 0x3)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/cmd/arena/commands/get.go:77 +0x2d5
github.com/kubeflow/arena/vendor/github.com/spf13/cobra.(*Command).execute(0xc000174500, 0xc00026ced0, 0x3, 0x3, 0xc000174500, 0xc00026ced0)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/kubeflow/arena/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc00002b680, 0xc000174280, 0xc000174000, 0xc000216f00)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/kubeflow/arena/vendor/github.com/spf13/cobra.(*Command).Execute(0xc00002b680, 0x2, 0xc0003b8c40)
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena/cmd/arena/main.go:59 +0x53

```
root@root1-HP-EliteBook-840-G2:/home/root1/SriniCH/workspace/src/github.com/kubeflow/arena# ./bin/arena version
arena: v0.2.0+44b0c1d
  BuildDate: 2019-06-18T11:55:21Z
  GitCommit: 44b0c1de82d5a6364f70bc27e3726ea6e29e9f64
  GitTreeState: clean
  GoVersion: go1.11.5
  Compiler: gc
  Platform: linux/amd64

--- a/cmd/arena/commands/trainer_spark.go
+++ b/cmd/arena/commands/trainer_spark.go
@@ -377,7 +377,7 @@ func (st *SparkJobTrainer) getTrainingJob(name, namespace string) (job TrainingJ
 		return nil, err
 	}
 	if len(sparkjobList.Items) == 0 {
-		return nil, nil
+		return nil, fmt.Errorf("Failed to find the job for %s", name)
 	} else {
 		sparkjob = sparkjobList.Items[0]
 	}

